draft-yasskin-httpbis-origin-signed-exchanges-impl.md

```diff
@@ -461,13 +460,16 @@ to retrieve an updated OCSP from the original server.
    certificate chain in its Certificate Verify (Section 4.4.3 of
    {{?I-D.ietf-tls-tls13}}), in order to allow updating the stapled OCSP
    response without updating signatures at the same time.
-1. If `signature` is a valid signature of `message` by `publicKey` using
-   `signing-alg`, return "potentially-valid" with `certificate-chain`.
-   Otherwise, return "invalid".
+1. If `signature` is not a valid signature of `message` by `publicKey` using
+   `signing-alg`, return "invalid".
+1. If `integrity` does not match "digest/mi-sha256-03", return "invalid".
```

Comment:
Why move this down here from line 419?

Reply:
We wanted to clarify that the UAs may start processing the headers part before verifying the entire payload.
Before this change, invalid payload could make headers invalid. After this change the headers checks are somewhat independent.

The original change was from @jyasskin, so I may be wrong.